entry_types/scrolled/package/src/frontend/playerControls/ProgressIndicators.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {DraggableCore} from 'react-draggable';
+import Measure from 'react-measure';
+import classNames from 'classnames';
+
+import styles from './ProgressIndicators.module.css';
+
+export class ProgressIndicators extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      progressBarsContainerWidth: null
+    };
+
+    this.measureProgressIndicatorsContainer = ({width}) => {
+      this.setState({progressBarsContainerWidth: width});
+    };
+
+    this.handleStop = (mouseEvent, dragEvent) => {
+      console.log('(STOP) seek to: '+ this.positionToTime(dragEvent.x));
+    };
+
+    this.handleDrag = (mouseEvent, dragEvent) => {
+      console.log('(DRAG) seek to: '+ this.positionToTime(dragEvent.x));
+    };
+  }
+
+  loadProgress() {
+    return this.props.duration > 0 ? (this.props.bufferedEnd / this.props.duration) : 0;
+  }
+
+  playProgress() {
+    return this.props.duration > 0 ? (this.props.currentTime / this.props.duration) : 0;
+  }
+
+  handlePosition() {
+    if (this.state.progressBarsContainerWidth) {
+      return ((this.state.progressBarsContainerWidth) * this.playProgress());
+    }
+    else {
+      return toPercent(this.playProgress());
+    }
+  }
+
+  positionToTime(x) {
+    if (this.props.duration && this.state.progressBarsContainerWidth) {
+      const fraction = Math.max(0, Math.min(1, x / this.state.progressBarsContainerWidth));
+      return fraction * this.props.duration;
+    }
+    else {
+      return 0;
+    }
+  }
+
+  render() {
+    return (
+      <div className={classNames(styles.progressIndicatorsContainer)}>
```

Comment:
Tabstop/keyboard handling is still missing. Can maybe be a checkbox item on an issue so we remember, when we make keyboard navigation of context menus work?

Reply:
I added a checkbox to the keyboard navigation ticket (https://redmine.codevise.de/issues/17392)